options.go

```diff
@@ -2,31 +2,22 @@ package gosundheit
 
 type Option func(*health)
 
-// WithCheckListener allows you to listen to check start/end events
-func WithCheckListener(listener CheckListener) Option {
+// WithCheckListeners allows you to listen to check start/end events
+func WithCheckListeners(listener ...CheckListener) Option {
 	return func(h *health) {
-		if listener != nil {
-			h.checksListener = listener
-		}
+		h.checksListener = listener
 	}
 }
 
-func withDefaultCheckListener() Option {
+// WithHealthListeners allows you to listen to overall results change
+func WithHealthListeners(listener ...HealthListener) Option {
 	return func(h *health) {
-		if h.checksListener == nil {
-			h.checksListener = noopCheckListener{}
-		}
+		h.healthListener = listener
 	}
 }
 
 // WithDefaults sets all the Health object settings. It's not required to use this as no options is always default
-// Defaults are: no check listener
+// This is a simple placeholder for any future defaults
```

Comment:
Suggestion: less code is better code. Drop this until you need it.

Reply:
It simplifies the service construction. Less code doesn't apply here.

Reply:
Well, it doesn't do anything, so I'd argue it doesn't simplify anything.